src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -143,8 +143,8 @@
         {% endif %}
     </div>
 
-    {% set symfony_status = { dev: 'In Development', stable: 'Maintained', eom: 'Unmaintained', eol: 'Unmaintained' } %}
-    {% set symfony_status_class = { dev: 'warning', stable: 'success', eom: 'error', eol: 'error' } %}
+    {% set symfony_status = { dev: 'In Development', stable: 'Maintained', eom: 'Unmaintained', eol: 'Expired' } %}
```

Comment:
On https://symfony.com/releases/3.4, the wording `Unmaintained` is used for an EOL version. We should be consistent here IMO.

I cannot check the wording for EOM versions on the website as there is no such version right now (4.4 is not yet EOM)

Reply:
I've checked symfony.com source and we use `Security fixes only` ... which is the same wording used by PHP in https://www.php.net/supported-versions.php So, I've updated the wording in this PR too. Thanks.